### PR TITLE
[Datasource] Workspace NSG Update

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Review by ismirlia — DISMISSED** [review]

File: website/docs/d/pi_workspace.html.markdown [hunk 1]
```diff
@@ -49,12 +55,15 @@ In addition to all argument reference listed, you can access the following attri
...
+  - `network_security_groups` - (List) Network security groups configuration.
+    Nested schema for `network_security_groups`:
+    - `state` - (String) The state of a network security groups configuration.
   - `power_edge_router` - (List) Power Edge Router information.
 
       Nested schema for `power_edge_router`:
-      - `migration_status` - (String) The migration status of a Power Edge Router.
-      - `status` - (String) The state of a Power Edge Router.
-      - `type` - (String) The Power Edge Router type.
+    - `migration_status` - (String) The migration status of a Power Edge Router.
```
Comment (ismirlia) [inline website/docs/d/pi_workspace.html.markdown:64]: I think the spacing is off here:
```suggestion
      - `migration_status` - (String) The migration status of a Power Edge Router.
```

File: website/docs/d/pi_workspace.html.markdown [hunk 2]
```diff
@@ -49,12 +55,15 @@ In addition to all argument reference listed, you can access the following attri
...
+    - `status` - (String) The state of a Power Edge Router.
```
Comment (ismirlia) [inline website/docs/d/pi_workspace.html.markdown:65]:
```suggestion
      - `status` - (String) The state of a Power Edge Router.
```

File: website/docs/d/pi_workspace.html.markdown [hunk 3]
```diff
@@ -49,12 +55,15 @@ In addition to all argument reference listed, you can access the following attri
...
+    - `type` - (String) The Power Edge Router type.
```
Comment (ismirlia) [inline website/docs/d/pi_workspace.html.markdown:66]:
```suggestion
      - `type` - (String) The Power Edge Router type.
```

File: website/docs/d/pi_workspaces.html.markdown [hunk 4]
```diff
@@ -31,38 +35,43 @@ Example usage:
...
+      - `status` - (String) The state of a Power Edge Router.
+      - `type` - (String) The Power Edge Router type.
   - `pi_workspace_id` - (String) Workspace ID.
   - `pi_workspace_location` - (Map) Workspace location.
 
       Nested schema for `Workspace location`:
-      - `region` - (String) Workspace location region zone.
-      - `type` - (String) Workspace location region type.
-      - `url`- (String) Workspace location region url.
+    - `region` - (String) Workspace location region zone.
```
Comment (ismirlia) [inline website/docs/d/pi_workspaces.html.markdown:72]: Same issue with this block

```suggestion
      - `region` - (String) Workspace location region zone.
```

File: website/docs/d/pi_workspaces.html.markdown [hunk 5]
```diff
@@ -31,38 +35,43 @@ Example usage:
...
+    - `type` - (String) Workspace location region type.
```
Comment (ismirlia) [inline website/docs/d/pi_workspaces.html.markdown:73]:
```suggestion
      - `type` - (String) Workspace location region type.
```

File: website/docs/d/pi_workspaces.html.markdown [hunk 6]
```diff
@@ -31,38 +35,43 @@ Example usage:
...
+    - `url`- (String) Workspace location region url.
```
Comment (ismirlia) [inline website/docs/d/pi_workspaces.html.markdown:74]:
```suggestion
      - `url`- (String) Workspace location region url.
```

File: website/docs/d/pi_workspace.html.markdown (hunk 3 above)
Comment (Alexander-Kita) [inline website/docs/d/pi_workspace.html.markdown:66]:
```suggestion
      - `migration_status` - (String) The migration status of a Power Edge Router.
      - `status` - (String) The state of a Power Edge Router.
      - `type` - (String) The Power Edge Router type.
```

File: website/docs/d/pi_workspaces.html.markdown [hunk 7]
```diff
@@ -31,38 +35,43 @@ Example usage:
...
+
 In addition to all argument reference listed, you can access the following attribute references after your data source is created.
 
 - `workspaces` - (List) List of all Workspaces.
   Nested schema for `workspaces`
   - `pi_workspace_capabilities` - (Map) Workspace Capabilities. Capabilities are `true` or `false`.
 
       Some of `pi_workspace_capabilities` are:
-      - `cloud-connections`, `power-edge-router`, `power-vpn-connections`, `transit-gateway-connection`
+    - `cloud-connections`, `power-edge-router`, `power-vpn-connections`, `transit-gateway-connection`
```
Comment (Alexander-Kita) [inline website/docs/d/pi_workspaces.html.markdown:52]:
```suggestion
      - `cloud-connections`, `power-edge-router`, `power-vpn-connections`, `transit-gateway-connection`
```

File: website/docs/d/pi_workspaces.html.markdown [hunk 8]
```diff
@@ -31,38 +35,43 @@ Example usage:
...
+    - `crn` - (String) Workspace crn.
+    - `network_security_groups` - (List) Network security groups configuration.
+      Nested schema for `network_security_groups`:
+      - `state` - (String) The state of a network security groups configuration.
+    - `power_edge_router` - (List) Power Edge Router information.
+
+      Nested schema for `power_edge_router`:
+      - `migration_status` - (String) The migration status of a Power Edge Router.
+      - `status` - (String) The state of a Power Edge Router.
+      - `type` - (String) The Power Edge Router type.
```
Comment (Alexander-Kita) [inline website/docs/d/pi_workspaces.html.markdown:67]:
```suggestion
      - `creation_date` - (String) Date of workspace creation.
      - `crn` - (String) Workspace crn.
      - `network_security_groups` - (List) Network security groups configuration.
      Nested schema for `network_security_groups`:
      - `state` - (String) The state of a network security groups configuration.
    - `power_edge_router` - (List) Power Edge Router information.

        Nested schema for `power_edge_router`:
        - `migration_status` - (String) The migration status of a Power Edge Router.
        - `status` - (String) The state of a Power Edge Router.
        - `type` - (String) The Power Edge Router type.
```

File: website/docs/d/pi_workspace.html.markdown [hunk 9]
```diff
@@ -7,22 +7,26 @@ description: |-
 ---
```
Comment (ismirlia) [inline website/docs/d/pi_workspace.html.markdown:7]: what's the consensus on 
````
page_title: "IBM: pi_workspace" 
```
vs 
```
page_title: "IBM: ibm_pi_workspace"
```

?

Comment (michaelkad) [reply website/docs/d/pi_workspace.html.markdown:7]: None -- We probably need to ask Harini about this later.

File: website/docs/d/pi_workspaces.html.markdown [hunk 10]
```diff
@@ -31,31 +35,37 @@ Example usage:
...
+
 In addition to all argument reference listed, you can access the following attribute references after your data source is created.
 
 - `workspaces` - (List) List of all Workspaces.
   Nested schema for `workspaces`
   - `pi_workspace_capabilities` - (Map) Workspace Capabilities. Capabilities are `true` or `false`.
 
       Some of `pi_workspace_capabilities` are:
-      - `cloud-connections`, `power-edge-router`, `power-vpn-connections`, `transit-gateway-connection`
+    - `cloud-connections`, `power-edge-router`, `power-vpn-connections`, `transit-gateway-connection`
```
Comment (Alexander-Kita) [inline website/docs/d/pi_workspaces.html.markdown:52]:
```suggestion
      - `cloud-connections`, `power-edge-router`, `power-vpn-connections`, `transit-gateway-connection`
```

**Review by ismirlia — APPROVED** [review]

**Comment by michaelkad** [comment]
Opened in main repo

**Pull request closed without merging**